src/ln/msgs.rs

```diff
@@ -488,6 +488,17 @@ pub enum HTLCFailChannelUpdate {
 	},
 }
 
+/// For events which result in both a RevokeAndACK and a CommitmentUpdate, by default they should
+/// be sent in the order they appear in the return value, however sometimes the order needs to be
+/// variable at rumtime. In those cases, this enum is also returned.
```

Comment:
*runtime

Reply:
I kinda prefer rum-time, but, ok, fixed :p

Reply:
Ah nop, need to get out the 0.1 before to get there ;)